packages/pytest-simcore/src/pytest_simcore/docker_compose.py

```diff
@@ -154,9 +156,46 @@ def simcore_docker_compose(
     return config
 
 
+@pytest.fixture(scope="module")
+def inject_filestash_config_path(
+    osparc_simcore_scripts_dir: Path,
+    monkeypatch_module: MonkeyPatch,
+    env_file_for_testing: Path,
+) -> None:
+    create_filestash_config_py = (
+        osparc_simcore_scripts_dir / "filestash" / "create_config.py"
+    )
+
+    # ensures .env at git_root_dir, which will be used as current directory
+    assert env_file_for_testing.exists()
+    env_values = dotenv_values(env_file_for_testing)
+
+    process = subprocess.run(
+        ["python3", f"{create_filestash_config_py}"],

```

Comment:
TIP: why not taking advantage that this is python and you just import it to get the path?